src/test/scala/sorm/test/general/EntityWithNoPrimitivesSuite.scala

```diff
@@ -24,7 +24,7 @@ class EntityWithNoPrimitivesSuite extends FunSuite with ShouldMatchers {
       .whereEqual("id", 1)
       .fetchOne()
       .map(a => a.copy(genres = Set("rock","rnb"), a = Set("myfoo"), b = Set("mybar")))
-      .map(db.save)
+      .map(a => db.save(a, false))
```

Comment:
Ok, changed to saveDelayed

On Thu, Mar 26, 2015 at 2:45 AM, Nikita Volkov <notifications@github.com>
wrote:

> In src/test/scala/sorm/test/general/EntityWithNoPrimitivesSuite.scala
> <https://github.com/sorm/sorm/pull/55#discussion_r27189701>:
>
> > @@ -24,7 +24,7 @@ class EntityWithNoPrimitivesSuite extends FunSuite with ShouldMatchers {
> >        .whereEqual("id", 1)
> >        .fetchOne()
> >        .map(a => a.copy(genres = Set("rock","rnb"), a = Set("myfoo"), b = Set("mybar")))
> > -      .map(db.save)
> > +      .map(a => db.save(a, false))
>
> @danicuki <https://github.com/danicuki>
> This indicates a broken backwards compatibility. Better introduce the
> feature as a saveDelayed method on Instance instead of the optional
> delayed parameter to the save method as it's done currently.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sorm/sorm/pull/55/files#r27189701>.
>



-- 
Daniel Cukier
Software Artist
+55 11 99991-3913